tutorials/get-started-kubernetes-loadbalancer/index.mdx

```diff
@@ -0,0 +1,180 @@
+content:
+  h1: Getting started with Kubernetes Part 2 - Load Balancers
+  paragraph: This is the third in a series of tutorials accompanying our video demonstrations on getting started with Kubernetes. Part 3 explains why you should create a Load Balacer for your Kapsule cluster, and walks you through how to achieve this.
+tags: Kubernetes k8s managed-kubernetes services loadbalancer load-balancer kapsule kubectl nodeport
+categories:
+  - compute
+dates:
+  validation: 2022-08-18
+  posted: 2022-08-18
+---
+
+This tutorial accompanies the third [video demonstration](https://www.youtube.com/watch?v=SVShdUO5fCw) in our series to help users get started with Kubernetes. We walk you through Kubernetes fundamentals for beginners. In this installment, we show you how to create a Load Balancer for your [Scaleway Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/).  
+
+First, we address the question of why we need a Load Balancer for our cluster, comparing it with the NodePort service we used in the previous tutorial. We then explain how to create a Load Balancer, check it out in the Scaleway console, and finally test that we can access our cluster's deployed application at the Load Balancer's IP address. 
+
+The next and final video in this series will look at the topic of persistent storage for a Kubernetes cluster.
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [generated your API Key](/console/my-project/how-to/generate-api-key/)
+  - You have followed the [first](/tutorials/get-started-containers-docker/) and [second](/tutorials/get-started-deploy-kapsule/) in this series, and have a Kubernetes Kapsule cluster running with the `whoami` application deployed on it.
+
+</Message>
+
+## Where were we?
+
+In our previous tutorials, we created a [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and deployed the containerized [whoami](https://github.com/jwilder/whoami) application to it. This is what our cluster looked like at the end of the last tutorial:
+
+<Lightbox src="scaleway-cluster.webp" alt="" />
+
+## Why do we need a Load Balancer?
+
+By default, Kubernetes clusters are not exposed to the Internet. These means that external users cannot access the application deployed in our cluster.
+
+In the previous tutorial, we solved this problem by creating a **NodePort** Service for our cluster. NodePort is a Kubernetes service which opens a port on each node in a cluster that contains pods for the specified deployment. It then forwards any external traffic received on that port to the right pods. This allowed us to go to our node’s external IP address in a browser, specifying the open port, and be served  our web application.
```

Comment:
the sentence I changed was a bit long, but maybe my suggestion doesnt make sense
```suggestion
In the previous tutorial, we solved this problem by creating a **NodePort** Service for our cluster. NodePort is a Kubernetes service which opens a port on each node in a cluster. This cluster contains pods for the specified deployment. It then forwards any external traffic received on that port to the right pods. This allowed us to go to our node’s external IP address in a browser, specifying the open port, and be served  our web application.
```

Reply:
This suggestion changes the meaning, but I've tried to improve the sentence a different way.